fairlearn/metrics/__init__.py

```diff
@@ -41,6 +41,7 @@
 from ._generated_metrics import _generated_metric_dict  # noqa: F401
 from ._make_derived_metric import make_derived_metric  # noqa: F401
 from ._metric_frame import MetricFrame  # noqa: F401
+from ._multi_curve_plot import plot_model_comparison  # noqa: F401
```

Comment:
We could rename the file `_multi_curve_plot` to `_plot_model_comparison` for clarity?

Reply:
@hildeweerts , @riedgar-ms -- do you have opinion on this? the `_multi_curve_plot` is probably just a historic leftover...